week-6/Homework/mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,39 @@
+// Global variables
+let quote = document.getElementById("quote"), 
+    author = document.getElementById("author"),
+    newbtn = document.getElementById("newbtn"),
+    label = document.getElementById("autolabel"),
+    checkbox = document.getElementById("checkbox"),
+    timer;
+
+// Event listener to call change function.
+newbtn.addEventListener("click", () => {
+change();
+});
```

Comment:
In theory, as `change` is a function, it could be also passed as a parameter.
`newbtn.addEventListener("click", change)`

Reply:
I got carried away with the arrow functions :D

Reply:
corrected this one.

Reply:
No worries, end of the day it did the same.